pages/api/delegates/delegates.ts

```diff
@@ -0,0 +1,83 @@
+/*
+
```

Comment:
shouldn't we replace api/delegates/index.ts?

Reply:
Ideally, yes. But I didn't want to break functionality for services relying on the current delegates API endpoint.

Maybe we can use api/delegates/index.ts for this one and rename the current one to api/delegates/legacy.ts while we give others time to adjust to the new workflow?